Source/FieldSolver/WarpX_QED_K.H

```diff
@@ -74,8 +74,9 @@ void warpx_hybrid_QED_push (
     amrex::Array4<amrex::Real> const& Bx, amrex::Array4<amrex::Real> const& By,
     amrex::Array4<amrex::Real const> const& Bz, amrex::Array4<amrex::Real> const& tmpEx,
     amrex::Array4<amrex::Real> const& tmpEy, amrex::Array4<amrex::Real> const& tmpEz,
-    amrex::Real dx, amrex::Real dy, amrex::Real dz, amrex::Real dt,
-    const amrex::Real xi_c2)
+    amrex::Array4<amrex::Real> const& Jx, amrex::Array4<amrex::Real> const& Jy,
+    amrex::Array4<amrex::Real> const& Jz, amrex::Real dx, amrex::Real dy,
```

Comment:
while you change this:
can you please make `dx`, `dy`, `dz`, and `dt` `const` parameters?